src/list.tsx

```diff
@@ -373,11 +374,29 @@ const hasControlProps = (
     return ['switch', 'checkbox', 'radioValue', 'iconButton'].some((prop) => obj[prop] !== undefined);
 };
 
+const getNodeText = (node: HTMLElement | null): string => {
+    const raw = node?.innerText || node?.textContent || '';
+
+    if (process.env.NODE_ENV === 'test') {
+        return (
+            raw
+                // Normalise whitespace sequences to a single space
+                .replace(/\s+/g, ' ')
+                // Insert space between "non-space character" + "Uppercase"
+                // Ex: "lineExtra" -> "line Extra", "1Extra" -> "1 Extra"
+                .replace(/([^ ])([A-ZÁÉÍÓÚÑ])/g, '$1 $2')
+                .trim()
+        );
+    }
+    return raw;
+};
+
 const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, ref) => {
     const titleId = React.useId();
     const isInverse = useIsInverseOrMediaVariant();
     const {
         asset,
+        assetAlt,
```

Comment:
instead of adding a new prop, I think we should get the `alt` text from the asset element. Something similar as the `getNodeText` logic